components/search/DateFilter.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <b-form inline>
+    <legend>
+      {{ name }}
```

Comment:
name will be a field name like `proxy_dcterms_issued`. It needs to be made human-readable and localised by way of adding a computed method like:
```js
      facetName() {
        return this.$t(`facets.${this.name}.name`);
      }
```